source/basic.tex

```diff
@@ -4728,9 +4728,8 @@
 \defn{value}, which is one discrete element of an
 \impldef{values of a trivially copyable type} set of values.
 \begin{footnote}
-The
-intent is that the memory model of \Cpp{} is compatible
-with that of ISO/IEC 9899 Programming Language C.
+The intent is that the memory model of \Cpp{} is compatible
+with that of \IsoCUndated{}.
```

Comment:
I think this is talking about the programming language, not about the ISO publication, so maybe
```suggestion
with that of the C programming language.
```

Reply:
Sounds good, thanks!